README.md

```diff
@@ -0,0 +1,22 @@
+## 8Dock v. 0.9.5
+Original Author: Jón Ragnarsson  
+Other original contributors: Michael Pfeiffer
```

Comment:
Let's remove poor Michael and stick him with the rest into the copyrights, wehen we have a package built via haikuporter.